datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -529,18 +572,24 @@ def service_check(self, name, status, tags=None, hostname=None, message=None, ra
         )
 
     def _log_deprecation(self, deprecation_key, *args):
+        # type: (str, *Any) -> None
```

Comment:
```suggestion
        # type: (str, *str) -> None
```

Reply:
Hmm… Don't agree. These `*args` are eventually used to interpolate the warning message — so technically they don't _have_ to be strings. (That said, we only ever pass strings currently, so your suggestion would _work_.)

Reply:
Well, nevermind, this isn't a general-purpose method and in practice we only have the `service_tag` deprecation that accepts the integration name as an interpolation variable, so… OK, changing to `str`.

Reply:
Addressed in 3a63e639b :+1: